### DISCUSSION TITLE
workflow error help

### DISCUSSION THREAD

**Discussion by kmnahidulislam** [community/community#66904] [Programming Help]
### Body

i am facing problem with workflow..look the photo...
github full file is here:
https://github.com/kmnahidulislam/problems.git
please give me full solution

![Screenshot 2023-09-12 234618](https://github.com/community/community/assets/144264609/1747c151-42b7-4b9a-92a6-195b6cd345f4)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by aregtech** [comment]
Hi @kmnahidulislam 
I guess, you didn't created the submodule. Here is the screenshot of your repo:

![image](https://github.com/community/community/assets/49253642/b5af9187-0695-490b-923d-6c45e3d74000)

1. If you move mouse over _developerFolio_, it does not show that it is a clickable link;
2. You don't have *.gitmodules* file in your repo, where it contains the information of the submodule location and the linked repo.

In contrast what you have, here is the example of [another repo with submodule](https://github.com/aregtech/areg-sdk):
![image](https://github.com/community/community/assets/49253642/e0147405-bddc-4f18-85ee-6ba72a24e90e)

And the file [*.gitmodules*](https://github.com/aregtech/areg-sdk/blob/master/.gitmodules) should contain information like this:
```
[submodule "thirdparty/googletest"]
	path = thirdparty/googletest
	url = https://github.com/google/googletest.git
```
I assume you forgot to commit the *.gitmodules* file to the repo. If not, check one more time whether you did in the right way. [Here is a very simple example](https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/How-to-add-submodules-to-GitHub-repos) how to add a submodule.

I hope, this helped 🙂

**Comment by vanquang202** [comment]
Why don't you look at the errors displayed in the workflow, I see the errors displayed are very complete for you to fix  
![image](https://github.com/community/community/assets/83849198/1e69ae6e-54cc-4099-b3a3-0baf463c4fdf)
